src/Native/Unix/System.Native/pal_networking.cpp

```diff
@@ -1687,6 +1687,14 @@ extern "C" Error SystemNative_Bind(intptr_t socket, uint8_t* socketAddress, int3
 
     int fd = ToFileDescriptor(socket);
 
+    // On Windows, Bind during TCP_WAIT is allowed.
+    // On Unix, we set SO_REUSEADDR to get the same behavior.
+    if (protocolType == PAL_PT_TCP)
+    {
+        int optionValue = 1;
+        setsockopt(fd, SOL_SOCKET, SO_REUSEADDR, &optionValue, sizeof(int));
```

Comment:
What if setsockopt fails?

Reply:
Currently it's silently ignored. I don't expect it to fail, but I wouldn't want to cause it to fail the bind either.

Reply:
Can you add a comment to highlight that we're ignoring any error on purpose, and why?

Reply:
Also, is doing this during bind the right place? i.e. why here rather than after the socket is created?  What's the expected behavior if, for example, after accept is used to get a connected socket on a particular port, that socket is then closed, and code then tries to listen on the same port that accepted socket was on?